test/Image_Tests/src/Image_Read_Spec.enso

```diff
@@ -12,18 +13,13 @@ fetch addr file =
     Process.run "curl" [addr, "--silent", "--output", file.path]
```

Comment:
This will download the file on each run, right?
Can we prepend existence check to avoid that?
```suggestion
    if file.exists.not then
        Process.run "curl" [addr, "--silent", "--output", file.path]
```